examples/snippets/onboarding.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	pubnub "github.com/pubnub/go"
+)
+
+func main() {
+
+	// Config
+	config := pubnub.NewConfig()
+	config.PublishKey = "pub-c-3ed95c83-12e6-4cda-9d69-c47ba2abb57e"   //"demo"   //"demo"
```

Comment:
Should we remove these from `examples` ?

Reply:
same keys, will be disbled

Reply:
Removed